docker/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG DART_SASS_SHA256SUM=2eed8cf8ec08e59e14c3801819aacc296f09524dad10af4a88641205
 
 # Apk
 ARG NODEJS_VERSION=12.21.0-r0
-ARG GIT_VERSION=2.26.2-r0
+ARG GIT_VERSION=2.26.3-r0
```

Comment:
Why did you go for `2.26.3-r0` instead of `2.31.0`?
What was breaking that this fixes?

Reply:
Good question. I had no reason beyond reading this output: https://app.circleci.com/pipelines/github/transcom/move.mil/860/workflows/d66fdd85-ffa4-40c0-b40c-6efa89b10561/jobs/8131

```text
 ---> Running in e0e13d2fbb46
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  git-2.26.3-r0:
    breaks: world[git=2.26.2-r0]
The command '/bin/sh -c apk add --latest --update --no-cache         oniguruma         zlib-dev         libpng-dev         libzip-dev         nodejs="$NODEJS_VERSION"         npm="$NODEJS_VERSION"         git="$GIT_VERSION"         && rm -rf /var/cache/apk/*' returned a non-zero code: 1
```

I like your approach better. I'll see if 2.31.0 works.

Reply:
Seems like that also doesn't work.

```text
 ---> Running in d037c804f43f
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  git-2.26.3-r0:
    breaks: world[git=2.31.0]
The command '/bin/sh -c apk add --latest --update --no-cache         oniguruma         zlib-dev         libpng-dev         libzip-dev         nodejs="$NODEJS_VERSION"         npm="$NODEJS_VERSION"         git="$GIT_VERSION"         && rm -rf /var/cache/apk/*' returned a non-zero code: 1
```

I'm going to see what happens if we remove the version requirement. I'm not sure what benefit that provides. If it isn't required, then we'll have more flexibility with the other version upgrades.